.dependabot/config.yml

```diff
@@ -0,0 +1,11 @@
+version: 1
```

Comment:
Does this work correctly on a branch or does this need to be on master?

Reply:
You're right, I think this has to be on master. Will add it there